.github/workflows/vast.yaml

```diff
@@ -1096,3 +1096,43 @@ jobs:
       - name: Run unit tests
         working-directory: python
         run: poetry run pytest
+      - name: Build package
+        working-directory: python
+        run: poetry build
```

Comment:
Same as below: Please turn this into a block with | for consistency; this makes it easier to spot run blocks when scrolling through the file.